### PR TITLE
Manual Deployment for JCenter

### PR DESCRIPTION
**Pull request by ahinchman1**
CI tasks include:
 - linting
 - unit testing
 - manual deployment

### Dev Notes

Since the deployment is manual, it will be necessary to add the following lines to your `local.properties`:

```
bintray.user=groupon-android-oss
bintray.apikey={API_KEY}
```

Kotlin linting/formatting is also available in the IDE itself:
Screen Shot 2020-07-02 at 12 00 38 PM

### Disclaimers, Explanations, etc. about the Code

 - In `bintray.gradle` we choose to read the aforementioned properties in `local.properties`, so in scripting we just create a fake one for the runner so it doesn't crash when building.

### CODE REVIEW

File: katmaps-library/install.gradle [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+                scm {
+                    connection 'https://github.com/groupon/KatMaps.git'
+                    developerConnection 'https://github.com/groupon/KatMaps.git'
+                    url 'https://github.com/groupon/KatMaps'
+                }
+            }
+        }
+    }
+}
```
Comment (garfiec) [inline katmaps-library/install.gradle:39]: Can you add a new line at the end of the gradle files?

File: katmaps-library/bintray.gradle [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Properties properties = new Properties()
+properties.load( new FileInputStream("local.properties"))
+
+// Bintray
+bintray {
+    user = properties.getProperty("bintray.user")
+    key = properties.getProperty("bintray.apikey")
+
+    pkg {
+        repo = 'jcenter' // TODO
```
Comment (garfiec) [inline katmaps-library/bintray.gradle:23]: Is this part finished?

File: .github/workflows/verify_pr.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+jobs:
+  ktlint:
+    name: Verify ktlint
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: "checkout"
+        uses: actions/checkout@v2
+
```
Comment (garfiec) [inline .github/workflows/verify_pr.yml:17]: I think we can remove this line

File: katmaps-library/install.gradle (hunk 1 above)
Comment (ahinchman1) [reply katmaps-library/install.gradle:39]: no problem!

File: katmaps-library/bintray.gradle (hunk 2 above)
Comment (ahinchman1) [reply katmaps-library/bintray.gradle:23]: good catch, yeah that should be done.

File: demo-app/src/main/java/com/groupon/katmaps/demo/MainActivity.kt [hunk 4]
```diff
@@ -63,11 +72,14 @@ class MainActivity : AppCompatActivity(R.layout.activity_main) {
...
-                padding = lastPadding
-            ), true)
+            map.moveCamera(
+                MapBounds.fromCenter(
+                    center = poi.position,
+                    radius = Length.fromMiles(5.0),
+                    padding = lastPadding
+                ),
+                true
+            )
```
Comment (ahinchman1) [inline demo-app/src/main/java/com/groupon/katmaps/demo/MainActivity.kt:82]: This was a result of the `formatKotlin` task! 

Screen Shot 2020-07-02 at 12 00 38 PM

File: katmaps-library/bintray.gradle (hunk 2 above)
Comment (ahinchman1) [reply katmaps-library/bintray.gradle:23]: So... yes, it should finished on this side. But remember, @miamiheat710 and I said this is going to be manual deployment for now, so what must happen next is that when you pull these changes down, you will then will go to your `local.properties` file and add the following:

```
bintray.user=groupon-android-oss
bintray.apikey={API_KEY}
```

Comment (ahinchman1) [reply katmaps-library/bintray.gradle:23]: This file is marked in .gitignore!

File: katmaps-library/install.gradle (hunk 1 above)
Comment (ahinchman1) [reply katmaps-library/install.gradle:39]: ya!

**Pull request merged**